# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xdg-desktop-portal-cosmic/hotfix-pointer.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xdg-desktop-portal-cosmic-epoch-1.0.0-beta.2/src/access.rs.vanilla	2025-10-20 20:16:21.272557797 +0200
+++ xdg-desktop-portal-cosmic-epoch-1.0.0-beta.2/src/access.rs	2025-10-20 20:16:28.220757565 +0200
@@ -151,7 +151,6 @@
                 layer: cosmic_client_toolkit::sctk::shell::wlr_layer::Layer::Top,
                 keyboard_interactivity:
                     cosmic_client_toolkit::sctk::shell::wlr_layer::KeyboardInteractivity::OnDemand,
-                pointer_interactivity: true,
                 anchor: cosmic_client_toolkit::sctk::shell::wlr_layer::Anchor::empty(),
                 output: IcedOutput::Active,
                 namespace: "access portal".to_string(),
--- xdg-desktop-portal-cosmic-epoch-1.0.0-beta.2/src/screenshot.rs.vanilla	2025-10-20 20:18:38.226523317 +0200
+++ xdg-desktop-portal-cosmic-epoch-1.0.0-beta.2/src/screenshot.rs	2025-10-20 20:18:43.841375815 +0200
@@ -885,7 +885,6 @@
                         id: *id,
                         layer: Layer::Overlay,
                         keyboard_interactivity: KeyboardInteractivity::Exclusive,
-                        pointer_interactivity: true,
                         anchor: Anchor::all(),
                         output: IcedOutput::Output(output.clone()),
                         namespace: "screenshot".to_string(),
